### ISSUE TITLE
A11y_VSCode_Untitled tab_HighContrast: In high contrast mode, focus is not visible on "untitled" tab

### ISSUE THREAD

**Issue by Roopesh-T** [microsoft/vscode#188127]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS; #A11yTCS; #A11ySev2;#BM-VisualStudioCode-Win32-Nov2021;#Visual Studio Code Client;#Win11;#FTP;#DesktopApp;

## Environment:
Visual Studio Code – Insider
Version: 1.81.0-insider (user Setup)
OS Version: Microsoft Windows 11 Enterprise (22621.1992)

## Pre Requisite:

Change system high contrast to aquatic.

## Repro Steps:

1. Open Visual Studio Code in your system
2. Tab to the explorer and press ctrl+ alt+windows+n to add a new text file
3. press ctrl+M to activate the tab navigation
4. use shift tab for the editor of the text file and move the close button of the untitled tab.
5. Use one more shift tab and check if the keyboard focus is visible on the screen.

## Actual Result:
In high contrast mode, focus is not visible on "untitled-1" tab

## Expected Result:
In high contrast mode, focus should be visible on "untitled-1" tab

## User Impact:
Keyboard users and low vision users will finds difficult while working if they are unable to see the focus visibility on "untitled-1" tab

## Attachments:

[188127_A11y_VSCode_Untitled tab_HighContrast In high contrast mode, focus is not visible on untitled tab.zip](https://github.com/microsoft/vscode/files/12078697/188127_A11y_VSCode_Untitled.tab_HighContrast.In.high.contrast.mode.focus.is.not.visible.on.untitled.tab.zip)

**Comment by isidorn** [comment]
@Roopesh-T this works just fine for me. What are you doing differently? 
Your attachment failed to upload. A simple screenshot or gif should be enough. Thanks!

![Screenshot 2023-07-18 at 09 37 48](https://github.com/microsoft/vscode/assets/1926584/b8295e06-b9a4-4835-ac05-fd6eb7d8d788)

**Comment by Roopesh-T** [comment]
@isidorn i have updated the attachment now can you please check once.

**Comment by isidorn** [comment]
Ok so repro steps:
1. High contrast theme
2. Focus on tab
3. The yellow border outline is already rendered because the file is open. So we can not render the focus on top of it. The tab is rendered the same way if there is focus or not

Assigning first to @daviddossett since this feels like a theme bug. We could fix by using a dotted line to indicate open tab, and when it gets focus that it gets a full line or something like that?

We have to fix this in the next 2-3 milestones due to a11y requirements.

**Comment by meganrogge** [comment]
@daviddossett I find these can get lost without a milestone 😄

**Comment by daviddossett** [comment]
One challenge I see here is how we handle active and focus states across active and inactive editors. Active, unfocused editors use a `dotted` tab outline in hc themes. Active, focused editors, use `solid`.

We could use `dashed` for the active and reserve solid for focused?

<img width="602" alt="CleanShot 2023-08-29 at 15 06 32@2x" src="https://github.com/microsoft/vscode/assets/25163139/2042e44c-c538-476b-aeb3-ef128f711912">

**Comment by meganrogge** [comment]
> We could use dashed for the active and reserve solid for focused?

This sounds like a good approach to me

**Comment by daviddossett** [comment]
I think I've landed on a good path forward here. Active, focused editors retain a solid border style. Explicitly focusing on the tab via keyboard nav will draw a dashed border. Focusing on the editor itself will restore the solid state:

https://github.com/microsoft/vscode/assets/25163139/72b1b149-a8ce-402e-b32e-aca8918755ea

**Comment by Roopesh-T** [comment]
closing the bug as the issue is fixed.
**GitHub**:#Closed;
[188127_Closed.zip](https://github.com/microsoft/vscode/files/12548289/188127_Closed.zip)
